app/templates/search/cases.hbs

```diff
@@ -6,17 +6,34 @@
       data-test-route-search-cases-toggle-decisions={{true}}
       @label={{t "search-in-decisions-only"}}
       @checked={{this.decisionsOnly}}
-      @onChange={{this.toggleDecisionsOnly}}
-    />
-  </div>
-  <div class="au-c-card au-o-box au-o-box--small">
-    <Auk::Label>{{t "archive"}}</Auk::Label>
-    <AuControlCheckbox
-      @label={{t "search-include-archived"}}
-      @checked={{this.includeArchived}}
-      @onChange={{this.toggleIncludeArchived}}
+      @onChange={{set this "decisionsOnly"}}
     />
   </div>
+  {{#if (user-may "manage-cases")}}
+    <div class="au-c-card au-o-box au-o-box--small">
+      <Auk::Label>{{t "archive"}}</Auk::Label>
+      <AuControlCheckbox
+        @label={{t "search-include-archived"}}
+        @checked={{this.includeArchived}}
+        @onChange={{set this "includeArchived"}}
+      />
+      <AuControlCheckbox
+        @label={{t "search-archived-only"}}
+        @checked={{this.archivedOnly}}
+        @onChange={{set this "archivedOnly"}}
+      />
+    </div>
```

Comment:
both options seem somewhat mutually exclusive, radiobuttons?

Reply:
They are not mutually exclusive but `archivedonly` implies `includearchived`, otherwise there will always be no results. So using a radiobutton seems like a good solution to me.